src/pages/home/library/SongList.tsx

```diff
@@ -144,22 +140,22 @@ export default function SongList({
                           maxWidth: { xs: "148px", sm: "auto" },
                         } }
                       >
-                        { row.title }
+                        { song.title }
                       </Box>
                     </Box>
                   </StyledTableCell>
                   <StyledTableCell
                     sx={ { display: { xs: "none", sm: "table-cell" } } }
                   >
-                    { row.genre }
+                    { song.genre }
                   </StyledTableCell>
                   <StyledTableCell align="right">
                     <Box sx={ { display: { xs: "none", sm: "inline" } } }>
-                      { row.createdAt.slice(0, 10) }{ " " }
+                      { song.createdAt.slice(0, 10) }{ " " }
                     </Box>
-                    <IconButton>
-                      <VerticalEllipsis />
-                    </IconButton>
+                    <Link to="edit-song" state={ { ...song } }>
```

Comment:
Replaced it to this since we aren't sure how we want to handle the navigation to the song yet. Once that is figured out we can delete this and apply a desired solution